cloudfront-dist/deployment/README.md

```diff
@@ -0,0 +1,15 @@
+# Welcome to your CDK TypeScript project!
+
+This is a blank project for TypeScript development with CDK.
```

Comment:
Can you personalize this readme ? 😃

Reply:
sure, i update the Readme however it might require another round 😉